libs/core/langchain_core/stores.py

```diff
@@ -123,3 +129,292 @@ async def ayield_keys(
 
 
 ByteStore = BaseStore[str, bytes]
+
+
+class LocalFileStore(ByteStore):
```

Comment:
feel pretty strongly this shouldn't be in core. Maybe a `langchain-fs` package. Or just in community.

Feels too easy of a target for vulnerabilities.

Reply:
langchain-fs works